### DISCUSSION TITLE
Is there any example of a third-person game in O3DE

### DISCUSSION THREAD

**Discussion by dnqbob** [o3de/o3de#6552] [Help Wanted]
I tried to make a character which receives keyboard input and turning&moving. And failing on make character turning (in transform) to correct direction.

Is there any example on this?

**Comment by lmbr-pip** [comment]
Two of the sample projects are third person:
* https://github.com/o3de/ChatCambrioleur
* https://github.com/o3de/o3de-multiplayersample (this is more a networking sample, it has some basic input controls)

The pong tutorial section on "Scripting the Player" could be useful here to:  https://o3de.org/docs/learning-guide/tutorials/first-project

It may be helpful to show how far you have with scripting input. Do you have the input and script canvas components on the entity to control? You have an inputs mapping file created?

**Reply by dnqbob** [reply]
I actually make a showcase in #6561, but encounter an input problem that I am unable to solve and I made the model turn instead of the whole player actor itself (I still want to make the whole actor turns considering the collision box of player actor is not symmetric in my game)

**Reply by dnqbob** [reply]
As for "ChatCambrioleur" the current version 2111 cannot compile it. Although I can have a look at its canvas on dealing with input

**Comment by dnqbob** [comment]
Although I still unable to turn the collision box in "transform" when "rigid body" on (is it a bug? :( ), 
now I use a probably more robust and better solution on the player actor's actions. 

Thanks @lmbr-pip 's instruction on this, I looked into the "ChatCambrioleur" and got some knowledge in need to modify my script. And I finally find [a asset which works normally in animation](https://hydrav.itch.io/standardanimationfbx) in O3DE (although its left arm still not normal in O3DE). 

![done1](https://user-images.githubusercontent.com/13763394/147729384-10835810-b381-4593-a7a0-99bdbd24a6b3.gif)
